### ISSUE TITLE
sftp upload function not work

### ISSUE THREAD

**Issue by GyunHo** [shaqian/flutter_ssh#22]
After creating an image file with imagepicker, I entered the path of the image file as the file path. But I get a No such file error.
The connection status and download function work.
But I don't know how to use the upload function.
How does it work?

**Comment by shaqian** [comment]
Hi @GyunHo,

I'm not sure what file path is returned from imagepicker. If download function works, you can set a breakpoint here
https://github.com/shaqian/flutter_ssh/blob/master/example/lib/main.dart#L136
get the value of `filePath`, then check if the file path from imagepicker is in the same format. 

Thanks,
Qian

**Comment by GyunHo** [comment]
> 안녕 @ 윤호 ,
> 
> 이미지 선택기에서 어떤 파일 경로가 반환되는지 확실하지 않습니다. 다운로드 기능이 작동하면 여기에서 중단 점을 설정할 수 있습니다
> https://github.com/shaqian/flutter_ssh/blob/master/example/lib/main.dart#L136
> 의 값을 얻은 `filePath`다음 imagepicker의 파일 경로가 같은 형식으로.
> 
> 고마워,
> Qian

Yes, if the download is successful, I think it should be possible to upload again based on the filepath.
But even if i run the example,  will get no Such File when uploading.
Am I doing something wrong ...? Does upload work as well as the example?
Thank you for your attention.

**Comment by GyunHo** [comment]
> Hi @GyunHo,
> 
> I'm not sure what file path is returned from imagepicker. If download function works, you can set a breakpoint here
> https://github.com/shaqian/flutter_ssh/blob/master/example/lib/main.dart#L136
> get the value of `filePath`, then check if the file path from imagepicker is in the same format.
> 
> Thanks,
> Qian

Yes, if the download is successful, I think it should be possible to upload again based on the filepath.
But even if i run the example, will get no Such File when uploading.
Am I doing something wrong ...? Does upload work as well as the example?
Thank you for your attention.

**Comment by shaqian** [comment]
Upload in the example app should work. 

Can you please share an example of the file path you got?

Thanks,
Qian

**Comment by IsaBass** [comment]
I am unable to connect to any ftp server. Have you been able to connect with the SSH package? Filezilla connects normally. My desktop application also connects to the server, but I have not been able to connect with the SSH package. The app freezes on the client.connect () command; without any error message.

**Comment by shaqian** [comment]
@IsaBass 
Are you able to debug the app in XCode or Android Studio? Please set a breakpoint in the connectToHost() function then step in to see where it gets stuck. 

For iOS:
https://github.com/shaqian/flutter_ssh/blob/master/ios/Classes/SshPlugin.m#L114
For Android:
https://github.com/shaqian/flutter_ssh/blob/master/android/src/main/java/sq/flutter/ssh/SshPlugin.java#L215

Thanks,
Qian
